index.html

```diff
@@ -2357,13 +2357,12 @@ <h4>Events API</h4>
                                     the payload of `thing_deleted` events when `diff` is set to `true`. 
                                 </li>
                                 <li>
-                                    <span class="rfc2119-assertion" id="tdd-notification-data-diff-unsupported">
-                                        When a server which does not support the `diff` query parameter 
-                                        is requested with such query parameter, it MUST
-                                        reject the request.
-                                    </span>
-                                    This is to inform the clients about the lack of such functionality at the
-                                    connection time to avoid runtime exceptions caused by missing
+                                    <!-- <span class="rfc2119-assertion" id="tdd-notification-data-diff-unsupported"> -->
+                                    If a server does not support the `diff` query parameter 
+                                    it is recommended that implementations reject such requests with an error.
```

Comment:
I think the original statement was clearer. After replacing "MUST" with "should':

```suggestion
                                        When a server which does not support the `diff` query parameter 
                                        is requested with such query parameter, it should reject the request.
```